pymchelper/readers/fluka.py

```diff
@@ -357,3 +361,38 @@ def get_unit_for_scoring(cls, scoring: str) -> str:
 
         # if unknown scoring, return empty string
         return ''
+
+
+@dataclass(frozen=True)
```

Comment:
SHIELD-HIT12A may also profit from such a class, maybe you should move it somewhere here ?
https://github.com/DataMedSci/pymchelper/blob/master/pymchelper/axis.py

Reply:
Moved and renamed